js/src/personal/tony/Tony.module.css

```diff
@@ -1,15 +1,29 @@
 .title {
-  color: light-dark(var(--mantine-color-black), var(--mantine-color-white));
-  font-size: rem(100px);
-  font-weight: 900;
-  letter-spacing: rem(-2px);
-
+  font-size: rem(60px);
+  font-weight: 400;
+  letter-spacing: rem(-1px);
+  color: white;
   @media (max-width: $mantine-breakpoint-md) {
     font-size: rem(50px);
   }
 }
-
-.background{
+.text {
+  color: white;
+  font-size: rem(30px);
+  font-weight: 200;
+  letter-spacing: rem(-1px);
```

Comment:
Negative letter spacing is dangerous, as it may make text hard to read.
It seems fine for the title, but you likely don't want it for the text or about.